.coveragerc

```diff
@@ -25,7 +25,7 @@ omit =
     *kombu/transport/zmq.py
     *kombu/transport/django.py
     *kombu/transport/pyro.py
-    *kombu/transport/azure*
+    *kombu/transport/azurestoragequeues.py
```

Comment:
Why did you exclude servicebus from test coverage?

Reply:
I removed it from the `omit` because it was not being generated his coverage report